clib/README.md

```diff
@@ -0,0 +1,12 @@
+# CLIB
+
+When a CLI needs a LIB, one gets a CLIB.
```

Comment:
must.. resist.. temptation to look up in urbandictionary

Reply:
Unintentional, but yeh... Best not hahaha